package.json

```diff
@@ -56,27 +56,27 @@
     "@bbc/gel-foundations": "^3.4.0",
     "@bbc/moment-timezone-include": "^1.1.2",
     "@bbc/psammead-amp-geo": "^1.1.2",
-    "@bbc/psammead-assets": "^2.4.0",
-    "@bbc/psammead-brand": "^5.0.3",
-    "@bbc/psammead-caption": "^2.2.11",
-    "@bbc/psammead-consent-banner": "^2.3.12",
-    "@bbc/psammead-copyright": "^1.2.10",
-    "@bbc/psammead-figure": "^1.2.5",
-    "@bbc/psammead-headings": "^3.1.11",
+    "@bbc/psammead-assets": "^2.4.1",
```

Comment:
Are all these package bumps related to this work?

Reply:
A fair amount of them are directly related, I decided to bump some of the other ones as well whilst I was at it but I probably should have made another pr for it instead. Would it be better if I just bump the things directly related to this?

Reply:
Yeh it's better to only make changes related to the issue you're solving and not lump in extra stuff while you're at it. If you do add extra stuff then mention it in the PR description so as not to confuse reviewers. Leave these changes in though.

Reply:
gotcha, will do in future.